test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -163,8 +166,7 @@ Describe "Set/New-Service cmdlet tests" -Tags "Feature", "RequireAdminOnWindows"
             [System.Management.Automation.PSCredential]::new("username", 
             (ConvertTo-SecureString "PlainTextPassword" -AsPlainText -Force)))
         },
-        # This test case fails due to #4803. Disabled for now.
-        # @{name = 'badstarttype'; parameter = "StartupType"; value = "System"},
+        @{name = 'badstarttype'; parameter = "StartupType"; value = "System"},
```

Comment:
Should we add "Boot" too? 
Should we check valid and invalid values from `System.ServiceProcess.ServiceStartMode`? If yes it is better refactor the tests. And for `Set-Service` too.

Reply:
will change